openassessment/data.py

```diff
@@ -114,15 +115,15 @@ def write_to_csv(self, course_id):
             # so we select AssessmentPart and follow the foreign key to the Assessment.
             parts = self._use_read_replica(
                 AssessmentPart.objects.select_related('assessment', 'option', 'option__criterion')
-                    .filter(assessment__submission_uuid=submission_uuid)
-                    .order_by('assessment__pk')
+                .filter(assessment__submission_uuid=submission_uuid)
```

Comment:
This is a pep8 rule that we don't have exceptions for. I was just cleaning up the file on my way through.